packages/concerto-ui-react/src/reactformvisitor.js

```diff
@@ -264,7 +265,7 @@ class ReactFormVisitor extends HTMLFormVisitor {
             return (<div className={style} key={field.getName()+'_wrapper'}>
                         <input type={this.toFieldType(field.getType())}
                             className={styles.input}
-                            value={new Date(value).toDatetimeLocal()}
+                            value={moment(value).format("YYYY-MM-DDThh:mm:ss")}
```

Comment:
Should this be
1. in local time with no timezone?
2. in local time with an explicit timezone?
3. in `Z`?

I am thinking 2. might be best for portability of the resulting data, and stability when it is used in other timezones.

Reply:
I have set utc timezone @jeromesimeon